test.py

```diff
@@ -0,0 +1,18 @@
+import networkx as nx
+import matplotlib.pyplot as plt
+import numpy as np
+
+n = 8
```

Comment:
The tests should not involve drawing (and hoping that a human can figure it out).
Each test should be a function inside `generators/tests/test_small.py`.  Take a look at those to see what's usual.

Use something like:
```python
def test_generalized_petersen_graph():
    G = nx.generalized_petersen_graph(8, 3)
    H = nx.moebius_kantor_graph()
    assert nx.is_isomorphic(G, H)

    # then add another assert statement...  or create a separate function if you prefer.
```